### ISSUE TITLE
Accessibility Fixes

### ISSUE THREAD

**Issue by shrinkray** [Compass-Detroit/hackmi26#22]
# Fix accessibility bugs

issues found in mobile menu and a few other component

## aria hidden focus
- Ensure aria-hidden elements are not focusable nor contain focusable elements. 
- Focusable content should have tabindex="-1" or be removed from the DOM

## Attribute aria-hidden is unnecessary for elements that have attribute hidden. 
- ThreeBackground.astro data-astro-source-loc="30:4"

## aria-label attribute must not be specified on any div element 
- unless the element has a role value other than caption, code, deletion, emphasis, generic, insertion, paragraph, presentation, strong, subscript, or superscript.
- EventDetails.astro data-astro-source-loc="82:14"

## The list role is unnecessary for element ul
- Footer.astro data-astro-source-loc="19:55"

## Tab navigation
- Event details date section is skipped
- Add skip link button for date and for map

## Other inconsistencies
- Check font size for text following a heading
- text highlights
- Update Shrinkray messaging and branding
